README.md

```diff
@@ -20,23 +20,31 @@
 </p>
 
 
-## Table of contents
+## 📖 Table of contents
 
 - <a href="#community">Community</a>
 - <a href="#contributors">Contributors</a>
+- <a href="#Contributing">Contributing</a>
+- <a href="#License">License</a>
 
-<h2 id="community">Community</h2>
+<h2 id="community">🚀 Community</h2>
 We have a discord where we discuss about out updates, features, requests, best practices, contributions and everything. Please join us.
 
 [Join the Envless Discord](https://discord.gg/3H4xkDDwWj)
 
-## 🛡️ License
+<h2 id="Contributing">🧰 Contributing</h2>
+
+Whether you are a community member or not, we would love your point of view! Feel free to first check out our
+- [contribution guidelines](https://github.com/envless/envless/blob/main/CONTRIBUTING.md) 
```

Comment:
```suggestion
- [Contribution Guidelines](https://github.com/envless/envless/blob/main/CONTRIBUTING.md) 
```
Lets make it `Contribution Guidelines` so it matches with `CONTRIBUTING.md` file's title